plugins/hls-brittany-plugin/test/testdata/BrittanyCRLF.formatted_range.hs

```diff
@@ -1,4 +1,4 @@
-foo ::   Int   -> String-> IO ()
+foo ::   Int   -> String-> IO ()
```

Comment:
Oops, this should not be formatted. Could you add this to the ignore file list?

Reply:
Huh, it looks like this file is already in the ignored list... You still cannot make a commit without this change?

Reply:
Will fix it, thank you for noticing!

Reply:
Fixed!